content/influxdb/clustered/install/deploy.md

```diff
@@ -9,6 +9,46 @@ menu:
 weight: 104
 ---
 
+Use Kubernetes and related tools to deploy your InfluxDB cluster.
+This guide provides instructions for deploying your InfluxDB cluster using the
+following tools:
+
+- **kubectl**: CLI for controlling the Kubernetes cluster manager
+- **kubit**: A Kubernetes controller that can render and apply jsonnet
+  templates based on the [kubecfg](https://github.com/kubecfg/kubecfg) jsonnet
+  tooling and framework
+
+InfluxDB Clustered uses an `AppInstance` Kubernetes custom resource (CR) to
+configure and deploy your InfluxDB Cluster.
+Installing a CR requires cluster-wide permissions and may cause `kubectl` to
+fail if you do not have those permissions in your cluster.
+
+`kubit` uses your local `kubectl` credentials to install CRs.
+This still requires you to have permissions to install cluster-wide resources
+(CRs), but doesn’t require `kubit` to be installed with those permissions
+in the cluster. 
```

Comment:
```suggestion
`kubit` uses your local `kubectl` credentials to install CRs.
This still requires you to have permissions to install cluster-wide resources
(CRs), but doesn’t require `kubit` to be installed with those permissions
in the cluster. 
```

I _sort of_ understand what this is trying to say, but I think it's quite confusing (partially because my explanation was poor beforehand).

In my opinion, this can likely be removed or rejigged to something to the effect of:

"Your local `kubectl` credentials will be used to install the `AppInstance` CRD. If you do not have those permissions available to you, an alternative is to install the package yourself. To do so, please install the [`kubit`](https://github.com/kubecfg/kubit#cli-tool) tool."

---

Below this I see there is mention of the CLI tool and `kubit local apply`, so I'm not sure exactly how this should be structured, but my thinking is either a quick mention to prepare an install OR simply linking down to the alternative if CRD installation is not possible.